src/neo/Oracle/OraclePolicyContract.cs

```diff
@@ -0,0 +1,230 @@
+                Value = BitConverter.GetBytes(1000)
+            });
+            return true;
+        }
+
+        /// <summary>
+        /// Oracle validator can delegate third party to operate Oracle nodes
+        /// </summary>
+        /// <param name="engine">VM</param>
+        /// <param name="args">Parameter Array</param>
+        /// <returns>Returns true if the execution is successful, otherwise returns false</returns>
+        [ContractMethod(0_01000000, ContractParameterType.Boolean, ParameterTypes = new[] { ContractParameterType.ByteArray, ContractParameterType.ByteArray }, ParameterNames = new[] { "consignorPubKey", "consigneePubKey" })]
+        private StackItem DelegateOracleValidator(ApplicationEngine engine, Array args)
+        {
+            ECPoint consignorPubKey = args[0].GetSpan().AsSerializable<ECPoint>();
+            ECPoint consigneePubKey = args[1].GetSpan().AsSerializable<ECPoint>();
+            UInt160 account = Contract.CreateSignatureRedeemScript(consignorPubKey).ToScriptHash();
+            if (!InteropService.Runtime.CheckWitnessInternal(engine, account)) return false;
+            StoreView snapshot = engine.Snapshot;
+            StorageKey key = CreateStorageKey(Prefix_Validator, consignorPubKey);
+            if (snapshot.Storages.TryGet(key) != null)
+            {
+                StorageItem value = snapshot.Storages.GetAndChange(key);
+                value.Value = consigneePubKey.ToArray();
+            }
+            else
+            {
+                snapshot.Storages.Add(key, new StorageItem
+                {
+                    Value = consigneePubKey.ToArray()
+                });
+            }
+            return true;
+        }
+
+        /// <summary>
+        /// Get current authorized Oracle validator.
+        /// </summary>
+        /// <param name="engine">VM</param>
+        /// <param name="args">Parameter Array</param>
+        /// <returns>Authorized Oracle validator</returns>
+        [ContractMethod(0_03000000, ContractParameterType.Array)]
+        private StackItem GetOracleValidators(ApplicationEngine engine, Array args)
+        {
+            return new Array(engine.ReferenceCounter, GetOracleValidators(engine.Snapshot).Select(p => (StackItem)p.ToArray()));
+        }
+
+        /// <summary>
+        /// Get current authorized Oracle validator
+        /// </summary>
+        /// <param name="snapshot">snapshot</param>
+        /// <returns>Authorized Oracle validator</returns>
+        public ECPoint[] GetOracleValidators(StoreView snapshot)
+        {
+            ECPoint[] oraclePubKeys = PolicyContract.NEO.GetValidators(snapshot);
+            for (int index = 0; index < oraclePubKeys.Length; index++)
+            {
+                var oraclePubKey = oraclePubKeys[index];
+                StorageKey key = CreateStorageKey(Prefix_Validator, oraclePubKey);
+                ECPoint delegatePubKey = snapshot.Storages.TryGet(key)?.Value.AsSerializable<ECPoint>(1);
```

Comment:
Why this 1?

Reply:
Bug

Reply:
Fix

Reply:
We should cover whole the file 
https://coveralls.io/builds/28790140/source?filename=src%2Fneo%2FSmartContract%2FNative%2FOracle%2FOraclePolicyContract.cs#L42

Reply:
I know.I am trying to do this.But these methods involve multiple signatures, which makes test cases very complicated.The default address of the consensus is written in the configuration file.
😂

Reply:
I can help you if you want

Reply:
Thank you very much.You can try writing UT for `setTimeOut `and `setPerRequestFee`.I write delegate UT.

Reply:
Done, check my ut, using `ManualWitness` it's easier to fake the consensus signatures.

Reply:
@doubiliu could you give me rights for modify your PR, i can't push my changes.